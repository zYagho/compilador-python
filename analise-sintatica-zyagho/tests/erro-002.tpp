inteiro: a
inteiro: b
inteiro: c[]
flutuante: d[10][]

inteiro principal()
    leia(a)
    escreva(b)
fim
