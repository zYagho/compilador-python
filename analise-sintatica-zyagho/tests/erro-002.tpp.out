ERR-SYN-INDICE
ERR-SYN-INDICE
WAR-SYN-GEN-SYNTAX-TREE
WAR-SYN-OUTPUT-FILE
WAR-SYN-ANA-SUCCESS
